src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/AnalyzerTest`1.cs

```diff
@@ -588,7 +588,6 @@ private Solution GetSolution((string filename, SourceText content)[] sources, (s
         /// strings.</returns>
         protected Project CreateProject((string filename, SourceText content)[] sources, (string filename, SourceText content)[] additionalFiles, ProjectState[] additionalProjects, MetadataReference[] additionalMetadataReferences, string language)
         {
-            language = language ?? language;
```

Comment:
Why the behavior change?

Reply:
actually I have no idea what this was trying to do before

Reply:
> actually I have no idea what this was trying to do before

Exactly 😂